src/screens/token/components/delegate-token/components/tabs/account-tab/styles.ts

```diff
@@ -0,0 +1,35 @@
+import { StyleSheet } from 'react-native';
+import { BASE_DIMENSION, normalize } from '../../../../../../../styles/dimensions';
+import { ITheme } from '../../../../../../../core/theme/itheme';
+
+export default (theme: ITheme) =>
+    StyleSheet.create({
+        container: {
+            flex: 1,
+            backgroundColor: theme.colors.appBackground,
+            paddingHorizontal: BASE_DIMENSION,
+            marginTop: BASE_DIMENSION * 3,
+            paddingBottom: BASE_DIMENSION * 5
+        },
+        text: {
+            fontWeight: 'bold',
+            fontSize: normalize(22),
+            lineHeight: normalize(28),
+            textAlign: 'center',
+            letterSpacing: 0.35,
```

Comment:
i think we should use normalise here. @NovacAndrei any thoughts ?

Reply:
Hmm true. Well we use in the entire app almost the same letterSpacing value. We should move this in the same place as BASE_DIMENSION and avoid hardcoding it any more. 

But I think I will refactor this in a separate PR because I would like to change in the entire app everything with letterSpacing.